spec.bs

```diff
@@ -172,15 +172,16 @@ are:
 1. If |interestGroup|'s [=interest group/estimated size=] is greater than 50 KB, then
   [=exception/throw=] a {{TypeError}}.
 1. Let |p| be [=a new promise=].
-1. Return |p| and run the following steps [=in parallel=]: (TODO: Enqueue the steps to a
-  [=parallel queue=] instead.)
+1. Let |queue| be the result of [=starting a new parallel queue=].
+1. [=parallel queue/enqueue steps|Enqueue the following steps=] to |queue|:

```

Comment:
Whether this should be parallel or not is under discussion on #488

Reply:
Just replied. We can keep the discussion there.